src/client/Microsoft.Identity.Client/Internal/Constants.cs

```diff
@@ -43,6 +43,7 @@ internal static class Constants
         public const string ManagedIdentityResourceId = "mi_res_id";
         public const string ManagedIdentityDefaultClientId = "system_assigned_managed_identity";
         public const string ManagedIdentityDefaultTenant = "managed_identity";
+        public const string CredentialEndpoint = "http://169.254.169.254/metadata/identity/credential";
```

Comment:
Should we rename this to `ImdsCredentialEndpoint`?

Reply:
Credential endpoint is not specific to IMDS